sound/soc/intel/boards/sof_sdw.c

```diff
@@ -356,6 +356,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					RT711_JD2 |
 					SOF_SDW_FOUR_SPK),
 	},
+	{
```

Comment:
please add a RPL separator between platforms, as we've done for TGL, ADL, SDCA before.

Reply:
Thanks, will update